### PR TITLE
Document some of the constants that are exposed to the end user

### PR DESCRIPTION
**Pull request by leotrs**
Self-explanatory. You can see the output by executing `make html` from the `docs/` folder.

### CODE REVIEW

**Comment by PgBiel** [comment]
looks cluttered... options:
1. separate each constant and docstring by an empty line;
2. use `#: <text>` after the constant definition instead of `""" """` below (this wouldn't be possible everywhere, but at least for some groups of constants)

**Comment by leotrs** [comment]
@PgBiel done!

File: manim/constants.py [hunk 1]
```diff
@@ -32,12 +37,25 @@ class MyText(Text):
...
+"""One unit step in the positive Y direction."""
+
 DOWN = np.array((0.0, -1.0, 0.0))
+"""One unit step in the negative Y direction."""
+
 RIGHT = np.array((1.0, 0.0, 0.0))
+"""One unit step in the positive X direction."""
+
 LEFT = np.array((-1.0, 0.0, 0.0))
+"""One unit step in the nevative X direction."""
```
Comment (Aathish04) [inline manim/constants.py:52]:
```suggestion
"""One unit step in the negative X direction."""
```

File: manim/constants.py [hunk 2]
```diff
@@ -32,12 +37,25 @@ class MyText(Text):
...
+
 IN = np.array((0.0, 0.0, -1.0))
+"""One unit step in the nevative Z direction."""
```
Comment (Aathish04) [inline manim/constants.py:55]:
```suggestion
"""One unit step in the negative Z direction."""
```

**Comment by kilacoda** [comment]
Any reason to have the comments below their respective constants?

**Comment by leotrs** [comment]
> Any reason to have the comments _below_ their respective constants?

It has to. See [here](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute).

**Comment by kilacoda** [comment]
> > Any reason to have the comments _below_ their respective constants?
> 
> It has to. See [here](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute).

Ok. Thanks for clarifying.

**Comment by kolibril13** [comment]
looks good!
Do we maybe want to add a blank line between the directions constants and the numerical values?
xx

**Comment by leotrs** [comment]
> Do we maybe want to add a blank line between the directions constants and the numerical values?

I wanted to, but this page is being generated automatically by `autosummary`. It would be a bit too much work right now to break it up into two, I think, without modifying the autosummary extension in non-trivial ways.

**Comment by kolibril13** [comment]
Ok, it's not worth to spend much time on this detail

**Comment by leotrs** [comment]
> Ok, it's not worth to spend much time on this detail

Agreed. Thanks for merging for now. I still want to go back and make that split tho ;)

**Pull request merged**